frontend/components/ui/Rating.tsx

```diff
@@ -23,14 +24,17 @@ export const Rating = (props: RatingProps) => {
             } else if (halfStarsValue >= i * 2 - 1) {
                appearance = RatingStarAppearance.Half;
             }
-            return <RatingStar key={i} appearence={appearance} />;
+            return (
+               <RatingStar key={i} size={props.size} appearence={appearance} />
```

Comment:
This isn't new, but I just noticed that the prop on RatingStart `appearence` is misspelt. The variable we pass is correct (appearance).